README.md

```diff
@@ -25,11 +25,10 @@ curl https://sh.phylum.io/ | sh -
 
 ### Install on macOS
 
-On macOS, we recommend installing phylum with [Homebrew](https://brew.sh/):
+On macOS, we recommend installing the Phylum CLI with [Homebrew](https://brew.sh/):
```

Comment:
The thinking behind the change was that "phylum" (no styling) was ambiguous and out of place since it wasn't capitalized. However, changing it to `phylum` (code styled) also didn't look quite right since the Formula changed from `phylum` to `phylum-cli`.

I'll take a note in the larger story around this transition and change this to `phylum` once the custom `phylum` Formula is removed and a `phylum` alias is created in the Homebrew core tap.

Reply:
> The thinking behind the change was that "phylum" (no styling) was ambiguous and out of place since it wasn't capitalized.

My point here is that I want to normalize referring to the tool simply as "phylum", with no capitalization or styling required... If we need code styling for clarity, my point is lost and we may as well just call it "the Phylum CLI"...